src/platform/utilities/environment/stagingDomains.js

```diff
@@ -9,17 +9,25 @@ if (__BUILDTYPE__ === 'preview') {
   currentEnv = 'preview';
 }
 
+if (__BUILDTYPE__ === 'production') {
+  currentEnv = 'www';
+}
+
 // This list also exists in script/options.js
 const domainReplacements = [
-  { from: 'www\\.va\\.gov', to: `${currentEnv}.va.gov` },
+  {
+    from: 'https://www\\.va\\.gov',
+    // use relative links on dev to accomodate localhost
+    to: currentEnv === 'dev' ? '' : `https://${currentEnv}.va.gov`,
+  },
   { from: 'https://www\\.myhealth\\.va\\.gov', to: mhvBaseUrl() },
 ];
 
 // This doesn't include preview because we want to redirect to staging urls
 // in preview
 const prodEnvironments = new Set(['production']);
 
-function replaceWithStagingDomain(href) {
+export function replaceWithStagingDomain(href) {
```

Comment:
Decided to reuse this- exported it and modified the reg ex behavior a little. It had a regex for processing www.va.gov links but it doesn't look like we were using that rule before this change.